### ISSUE TITLE
Missing final newline in .zsh_plugins.txt causes last plugin to not load

### ISSUE THREAD

**Issue by Schmale97** [mattmc3/antidote#26]
I having this weird issue where antidote seems to be skipping zsh-users/zsh-history-substring-search into the ~/.zsh_plugins.zsh and I can't figure out why

Platform MacOs 12.4

Command
`antidote bundle < ~/.zsh_plugins.txt > ~/.zsh_plugins.zsh`

Contents of .zsh_plugins.txt
```
zsh-users/zsh-syntax-highlighting
zsh-users/zsh-completions
zsh-users/zsh-autosuggestions
zsh-users/zsh-history-substring-search
```

Contents of .zsh_plugins.zsh
```
fpath+=( /Users/username/Library/Caches/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-zsh-users-SLASH-zsh-syntax-highlighting )
source /Users/username/Library/Caches/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-zsh-users-SLASH-zsh-syntax-highlighting/zsh-syntax-highlighting.plugin.zsh
fpath+=( /Users/username/Library/Caches/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-zsh-users-SLASH-zsh-completions )
source /Users/username/Library/Caches/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-zsh-users-SLASH-zsh-completions/zsh-completions.plugin.zsh
fpath+=( /Users/username/Library/Caches/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-zsh-users-SLASH-zsh-autosuggestions )
source /Users/username/Library/Caches/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-zsh-users-SLASH-zsh-autosuggestions/zsh-autosuggestions.plugin.zsh
```

**Comment by mattmc3** [comment]
My suspicion is that you're missing a final newline at the end of your .zsh_plugins.txt file. That's definitely a bug in antidote, so I'll work on a patch.

In the meantime, run this to add the newline to your plugins file and it should resolve the issue:

```
echo >> ~/.zsh_plugins.txt
```

**Comment by Schmale97** [comment]
Yep, that fixed it :) thanks for the quick resposne

**Comment by mattmc3** [comment]
Fixed in 1.2, which was just released. Run `antidote update` to get 1.2.
